IN-CORE/incore-playbook
# Add support of earthquake tsunami hurricane and flood #44

Add all hazards support: eq, tsunami, tornado, hurricane and flood

**To test locally:**
1. npm install
2. npm install react@~16.14.0 react-dom@~16.14.0 react-redux@~7.1.0 redux-thunk@~2.3.0 redux@~4.0.1 react-router-dom@~5.2.0 @material-ui/core@^4.11.0 @material-ui/icons@^4.9.1 babel-polyfill@^6.26.0 --no-save
3. export DEPLOY_ENV=development
4. npm run
You should be able to uncomment `map.config.js` to set the map bounding box to different area.

**To test in child repo:**
0. `yarn link` to register the incore-playbook
1. Go to Joplin repo and checkout the latest main branch: https://github.com/IN-CORE/Joplin-Playbook
2. npm install
3. `yarn link incore-playbook`
4. export DEPLOY_ENV=development
You should be able to set map configuration and allowed hazard types here: https://github.com/IN-CORE/Joplin-Playbook/blob/main/src/index.jsx
